.github/workflows/test.yml

```diff
@@ -71,6 +71,25 @@ jobs:
           java --version
           native-image.cmd --version
         if: runner.os == 'Windows'
+  test-graalvm-ee:
+    name: GraalVM EE on ubuntu-latest
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run setup-graalvm action
+        uses: ./
+        with:
+          version: '21.3.0'
+          gds-token: ${{ secrets.GDS_TOKEN }}
+          java-version: '17'
+          # components: 'native-image' (activate after 22.1.0 is released)
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+      - name: Check environment
+        run: |
+          echo "GRAALVM_HOME: $GRAALVM_HOME"
+          echo "JAVA_HOME: $JAVA_HOME"
+          java --version
+          # native-image --version
```

Comment:
- [ ] This and the commented line above should be used after 22.1.0 is released.